action/install_test.go

```diff
@@ -0,0 +1,79 @@
+package action
+
+import (
+	"bytes"
+	"errors"
+	"os"
+	"testing"
+
+	"github.com/helm/helm/kubectl"
+	"github.com/helm/helm/log"
+)
+
+func TestInstall(t *testing.T) {
+	var output bytes.Buffer
+	log.Stdout = &output
+	log.Stderr = &output
+
+	// Todo: add tests
+	// - with an invalid chart name
+	// - with failure to check dependencies
+	// - with failure to check dependencies and force option
+	// - with chart in current directly
```

Comment:
there are enough todos here to warrant a new GH issue. not blocking this PRs merge